Applications/neighbourhoodstats/src/Highcharts/exporting-server/java/highcharts-export/highcharts-export-convert/src/main/java/com/highcharts/export/converter/SVGConverter.java

```diff
@@ -18,7 +18,7 @@
 import java.util.concurrent.TimeoutException;
 
 import org.apache.commons.io.FileUtils;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.Logger;
```

Comment:
used global replace based on this official guide: https://logging.apache.org/log4j/2.x/manual/migration.html#Log4j2API